data/data/bootstrap/files/usr/local/bin/bootstrap-in-place.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# This script is executed by bootkube.sh when installing single node with bootstrap in place
+
+if [ ! -f create-static-pods-data.done ]
+then
+    echo "Stop etcd static pod by moving the manifest"
+    mv /etc/kubernetes/manifests/etcd-member-pod.yaml /etc/kubernetes
+
+    until ! crictl ps | grep etcd
+    do
+        echo "Waiting for etcd to go down"
+        sleep 10
+    done
+
+    echo "Creating master ignition and writing it to disk"
+    # Get the master ignition from MCS
+    curl -k  -H "Accept:'application/vnd.coreos.ignition+json;version=3.1.0, */*;q=0.1'" \
+    https://localhost:22623/config/master -o /opt/openshift/master.ign
+
+    echo "Updating master ignition"
```

Comment:
with.... so next person who gonna look into it will understand the magic that is happening